CODIGO/frmCreateBattleground.frm

```diff
@@ -417,23 +460,27 @@ On Error GoTo ErrHandler:
     Settings.TeamSize = Val(tSize.Text)
     If Settings.MinPlayers Mod Settings.TeamSize <> 0 Or Settings.MaxPlayers Mod Settings.TeamSize <> 0 Then
         Call MsgBox(JsonLanguage.Item("MENSAJE_LIMITE_JUGADORES_DIVISIBLE"), vbExclamation)
-
         tSize.SetFocus
         Exit Sub
     End If
     
     Select Case cmbTipo.ListIndex
         Case 0
-            Settings.ScenearioType = 3
+            Settings.ScenearioType = e_EventType.DeathMatch
         Case 1
-            Settings.ScenearioType = 4
+            Settings.ScenearioType = e_EventType.NavalBattle
+        Case 2
+            Settings.ScenearioType = e_EventType.NpcHunt
+        Case 3
+            Settings.ScenearioType = e_EventType.CaptureTheFlag
     End Select
     
     Select Case cmbEquipos.ListIndex
         Case 0
```

Comment:
Change harcoded 0 for the correct value in e_TeamTypes

Reply:
en ves de
        Case 3
            Settings.ScenearioType = e_EventType.CaptureTheFlag
            
hacer:
        Case e_EventType.CaptureTheFlag  
            Settings.ScenearioType = e_EventType.CaptureTheFlag